examples/next-prisma-starter/src/pages/_app.tsx

```diff
@@ -17,14 +17,14 @@ function getBaseUrl() {
   if (process.browser) {
     return '';
   }
-  // // reference for vercel.com
+  // reference for vercel.com
   // if (process.env.VERCEL_URL) {
-  //   return process.env.VERCEL_URL;
+  //   return `https://${process.env.VERCEL_URL}`;
   // }
 
   // // reference for render.com
-  // if (process.env.RENDER_INTERNAL_HOSTNAME) {
-  //   return `http://${process.env.RENDER_INTERNAL_HOSTNAME}:${process.env.PORT}`;
+  // if (process.env.RENDER_EXTERNAL_URL) {
+  //   return process.env.RENDER_EXTERNAL_URL;
```

Comment:
This forces the server-side request to go through the interwebz in order to make a call to a server that sits in the same machine

Reply:
Also might not work for PR previews, but not sure

Reply:
I did not understand that. Will add a comment, ok?

Reply:
Just revert the render bit, I'm happy with the vercel change

Reply:
I use the render internal env var in my prod app